### DISCUSSION TITLE
Support for mowing areas and maintenance points

### DISCUSSION THREAD

**Discussion by ripvega** [Thomas55555/husqvarna_automower#358] [Ideas]
I have two 550 EPOS machines and they work well with the HA integration.

Being EPOS, they support different mowing areas and maintenance points. Unfortunately, the Husqvarna app doesn't support automating sending the mowers to maintenance points on a schedule. HA automation would be great for this.

I can't see maintenance points in the integration. Is this something that could be added?

I don't yet have the skills to do that, but can certainly provide the info needed.

**Comment by Thomas55555** [comment]
Hi,
these functions are not supported by the API. You can see the supported functions here:
https://developer.husqvarnagroup.cloud/apis/automower-connect-api#openapi

**Comment by ripvega** [comment]
Thank you.

I’ll email Husqvarna and see what they can add to the API.

**Reply by Din-BH** [reply]
Did you receive any feedback from Husqvarna team on this?
